policyengine_us/variables/gov/usda/csfp/commodity_supplemental_food_program.py

```diff
@@ -0,0 +1,14 @@
+from policyengine_us.model_api import *
+
+
+class commodity_supplemental_food_program(Variable):
+    value_type = float
+    entity = Person
+    unit = USD
+    definition_period = YEAR
```

Comment:
should this be monthly?

Reply:
Would that change the value of `wic_fpg`?

Reply:
Yeah let's keep as is until we tackle https://github.com/PolicyEngine/policyengine-us/issues/3689